gui/default/index.html

```diff
@@ -358,11 +358,11 @@ <h4 class="panel-title">
                         <a href="" ng-click="showFailed(folder.id)">{{model[folder.id].pullErrors | alwaysNumber | localeNumber}}&nbsp;<span translate>items</span></a>
                       </td>
                     </tr>
-                    <tr ng-if="folder.type != 'readwrite'">
+                    <tr ng-if="folder.type != 'sendreceive'">
                       <th><span class="fa fa-fw fa-lock"></span>&nbsp;<span translate>Folder Type</span></th>
                       <td class="text-right">
-                        <span ng-if="folder.type == 'readonly'" translate>Send Only</span>
-                        <span ng-if="folder.type != 'readonly'">{{ folder.type.charAt(0).toUpperCase() + folder.type.slice(1) }}</span>
+                        <span ng-if="folder.type == 'sendonly'" translate>Send Only</span>
+                        <span ng-if="folder.type == 'sendreceive'" translate>Send & Receive</span>
```

Comment:
you can drop this line as it will never show because <tr> above is conditional, which I think is best solution to save vertical space if folder runs in normal mode. No need to output "send & receive"